docs/openapi/tools-api.yaml

```diff
@@ -145,3 +145,44 @@ import-job-result:
         $ref: './responses.yaml#/404'
       '500':
         $ref: './responses.yaml#/500'
+import-assistant:
+  post:
+    tags:
+      - import
+    summary: Send a prompt to the import assistant
+    description: |
+      Note: Not implemented yet.
+      
+      This endpoint can be used to send prompts to the import assistant. 
+      Each prompt will make use of AI to analyze a URL and return
+      choices in response to the request.
+    operationId: processImportAssistant
+    security:
+      - api_key: [ ]
```

Comment:
possibly the list of available authentication methods (scoped api key, ims token, ...) needs to be updated and the appropriate one be selected here.

Reply:
Requests to Firefall need to be made from a service client. Our current plan would be to use a scoped api key to access this feature and then make Firefall requests using an accessToken from the spacecat Firefall client but include the userid and ims org id associated with the api key as part of the request.

Reply:
that's fine - all i am saying is, the endpoint security documented above should use something that is not deprecated :)

Reply:
HI @solaris007,
Anthony's on PTO for a couple weeks and I'll try to keep this PR moving.
From what I understand, you are saying `api_key` is deprecated and it should be something like `scoped_api_key`...  Is that correct?  It see `security: - api_key: [ ]` used in other APIs so I'm hesitating (maybe grandfathered?).